config/locales/en.yml

```diff
@@ -578,7 +578,7 @@ en:
       use_map_link: Use Map
     index:
       title: "Users' Diaries"
-      title_followed: "Followings' Diaries"
+      title_followed: "Your Followed Users' diaries"
```

Comment:
I think this could just be `Followed Users' diaries` which aligns well with the nearby users phrasing.

Reply:
Done in another commit. Thank you!